libs/ui-toolkit/src/components/slider/percentage-slider.tsx

```diff
@@ -86,9 +88,11 @@ export const PercentageSlider = ({
                     }
                   : undefined
               }
-              className="absolute translate-x-[-50%] text-xs font-alpha mt-1 text-vega-clight-100 dark:text-vega-cdark-100"
+              className={classNames(
+                'first:left-[1%] last:left-[98%] md:last:left-[99%] lg:last:left-[97%] absolute translate-x-[-50%] text-xs font-alpha mt-1 text-vega-clight-100 dark:text-vega-cdark-100'
```

Comment:
I think theres a better way to do this. This looks like a pain to maintain

Reply:
The way the scroll area for the deal ticket was set up wasn't super great which resulted in the cropping. We can just edit the padding of the scroll container, and avoid all the complexity of ensuring all the numbers line up with the slider indicators

```
const ContentWrapper = ({
  children,
  title,
}: {
  children: ReactNode;
  title?: string;
}) => {
  return (
    <TinyScroll className="py-4" data-testid="sidebar-content">
      {title && <h2 className="mb-4 px-3">{title}</h2>}
      <div className="px-3">{children}</div>
    </TinyScroll>
  );
};

```

Reply:
updated